less/mixins.less

```diff
@@ -1 +1,62 @@
-// Mixins in here
\ No newline at end of file
+/* Display Mixins */
+.flex(@direction: row, @justify: flex-start, @align: stretch) {
+    display: flex;
+    flex-direction: @direction;
+    justify-content: @justify;
+    align-items: @align;
+}
+
+/* Component Mixins */
+.btn-styles(@width: 83%) {
+    width: @width;
+    height: 45px;
+    border: 1px solid @button-border;
+    border-radius: 15px;
+    
+    font-family: @button-font;
+    font-size: 1.8rem;
+    color: @white;
+
+    background-color: @button-bg;
+
+    &:hover {
+        background-color: darken(@button-bg, 20%);
+    }
+}
+
+/* Media Query Mixins */
+.content-tablet(@contentClass: 'default') {
+    padding: 0 2rem;
+
+    & when (@contentClass = '.content-pick') {
+        padding: 3rem 2rem;
+
+        .destination .btn {
+            width: 100%;
+        }
+    }
+}
+
+.content-mobile(@newDirection, @contentClass: 'content-section') {
```

Comment:
As you have realized, putting parameters with default values after required ones is good practice, as you're less likely to be forced to supply an argument to a parameter with a default value just so you can supply an argument to the parameter you want. Nice work.